test/unit/tag_test.rb

```diff
@@ -294,4 +294,9 @@ def setup
     assert lat.location_tag?
     assert lon.location_tag?
   end
+
+  test 'Tag.get_recommendations' do
```

Comment:
OK, here's the unit test. Let's ensure it returns something! We could even test how many nodes it returns, or something else. But most importantly that it returns nodes at all!

Reply:
OK, see below where I made a further suggestion about the unit test: https://github.com/publiclab/plots2/pull/11332#discussion_r964248316